test/data/endpoints.json

```diff
@@ -0,0 +1,18 @@
+{
+  "endpoints": {
+    "v1": {
+      "version": "1.1.2",
+      "signalk-http": "http://192.168.1.2/signalk/v1/api/",
+      "signalk-ws": "ws://192.168.1.2:34567/signalk/v1/stream",
+      "signalk-tcp": {
```

Comment:
Re my comment above "I think we should use it anyway.." I meant lets use ```tcp://localhost:55555```.
As Teppo says - it keeps the format std, and we are going to get lots more in the future udp/rdp/mqtt/stomp/coap/thread, many of which will be unofficial